### PR TITLE
Implemented tests for creating new pages. Also added missing verifies.

### PR DESCRIPTION
**Pull request by marijnvanderhorst**
Added tests for creating new pages.

### CODE REVIEW

File: app/Http/Controllers/PaginaBeheerController.php [hunk 1]
```diff
@@ -95,6 +95,8 @@ private function validateData(Request $reqeust){
             'EN_text' => 'required',
             'content_nl' => 'required',
             'content_en' => 'required',
+            'parentItem' => 'required',
```
Comment (nieka) [inline app/Http/Controllers/PaginaBeheerController.php]: Als een menu item in de menu balk moet komen te staan dan heeft die geen parentItem zo dit veld kan niet verplicht zijn.

File: app/Http/Controllers/PaginaBeheerController.php [hunk 2]
```diff
@@ -95,6 +95,8 @@ private function validateData(Request $reqeust){
...
+            'afterItem' => 'required',
```
Comment (nieka) [inline app/Http/Controllers/PaginaBeheerController.php]: Bij het menu kan er ook gekozen worden om een menu item niet ergens na te plaatsen zo deze kan ook niet verplicht zijn.

Daarnaast in het dropdown menu is het eerst item ook niet na iets

Comment (marijnvanderhorst) [reply app/Http/Controllers/PaginaBeheerController.php]: De repository gaat er altijd vanuit dat `afterItem` bestaat. Indien `itemType` gelijk is aan `subItem`, moet de `parentItem` ook defined zijn. Ik heb dit aangepast in mijn laatste commit. Ook heb ik nog een test case toegevoegd om alle edge cases hiervan te testen.

File: app/Http/Controllers/PaginaBeheerController.php (hunk 1 above)
Comment (marijnvanderhorst) [reply app/Http/Controllers/PaginaBeheerController.php]: Aangepast dat die alleen verplicht is als `itemType` gelijk is aan `subItem`. Overigens stuurt de front-end dit veld altijd mee.
